cmd/qat_plugin/dpdkdrv/dpdkdrv.go

```diff
@@ -143,6 +216,36 @@ func (dp *DevicePlugin) Scan(notifier dpapi.Notifier) error {
 	}
 }
 
+// Implement the PreferredAllocator interface.
+func (dp *DevicePlugin) GetPreferredAllocation(rqt *pluginapi.PreferredAllocationRequest) (*pluginapi.PreferredAllocationResponse, error) {
+	response := &pluginapi.PreferredAllocationResponse{}
+
+	for _, req := range rqt.ContainerRequests {
+		klog.V(3).Infof("AvailableDeviceIDs: %q", req.AvailableDeviceIDs)
+		klog.V(3).Infof("MustIncludeDeviceIDs: %q", req.MustIncludeDeviceIDs)
+		klog.V(3).Infof("AllocationSize: %d", req.AllocationSize)
```

Comment:
a bit too much logging, IMO (the same for the `dp.policy()` functions). How about just one line after `dp.policy()` that logs, e.g., `req.AvailableDeviceIDs` and selected `DeviceIDs`?